libavfilter/dnn/dnn_backend_tf.c

```diff
@@ -72,6 +89,29 @@ static void free_buffer(void *data, size_t length)
     av_freep(&data);
 }
 
+static void tf_free_request(TFInferRequest *request)
+{
+    if (!request)
+        return;
+    if (request->input_tensor) {
+        TF_DeleteTensor(request->input_tensor);
+        request->input_tensor = NULL;
+    }
+    av_freep(&request->tf_input);
+    av_freep(&request->tf_outputs);
+    av_freep(&request->output_tensors);
```

Comment:
the below are missed?

TF_DeleteTensor(request->output_tensors[i]);
av_freep(&request);

Reply:
Oh, yes! TF_DeleteTensor for output tensors is missed. I'll add it.

Actually, this function frees only the contents of the request (used in the completion callback). So the same request item can be used again after inference. That's why the request pointer is not freed.